### ISSUE TITLE
GoogleAuthException: InternalError

### ISSUE THREAD

**Issue by DenBond7** [FlowCrypt/flowcrypt-android#245]
```
[STACK_TRACE]
com.flowcrypt.email.util.exception.ManualHandledException: Handled manually:
        at com.flowcrypt.email.api.email.sync.EmailSyncManager$BaseSyncRunnable.runSyncTask(EmailSyncManager.java:496)
        at com.flowcrypt.email.api.email.sync.EmailSyncManager$ActiveSyncTaskRunnable.run(EmailSyncManager.java:574)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:428)
        at java.util.concurrent.FutureTask.run(FutureTask.java:237)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
        at java.lang.Thread.run(Thread.java:762)
Caused by: com.google.android.gms.auth.GoogleAuthException: InternalError
        at com.google.android.gms.auth.zze.zzac(Unknown Source)
        at com.google.android.gms.auth.zzd.zza(Unknown Source)
        at com.google.android.gms.auth.zzd.zzb(Unknown Source)
        at com.google.android.gms.auth.zzd.getToken(Unknown Source)
        at com.google.android.gms.auth.zzd.getToken(Unknown Source)
        at com.google.android.gms.auth.GoogleAuthUtil.getToken(Unknown Source)
        at com.flowcrypt.email.api.email.protocol.OpenStoreHelper.openAndConnectToGimapsStore(OpenStoreHelper.java:77)
        at com.flowcrypt.email.api.email.protocol.OpenStoreHelper.openAndConnectToStore(OpenStoreHelper.java:172)
        at com.flowcrypt.email.api.email.sync.EmailSyncManager$BaseSyncRunnable.openConnectionToStore(EmailSyncManager.java:446)
        at com.flowcrypt.email.api.email.sync.EmailSyncManager$BaseSyncRunnable.runSyncTask(EmailSyncManager.java:475)
        ... 6 more
com.google.android.gms.auth.GoogleAuthException: InternalError
        at com.google.android.gms.auth.zze.zzac(Unknown Source)
        at com.google.android.gms.auth.zzd.zza(Unknown Source)
        at com.google.android.gms.auth.zzd.zzb(Unknown Source)
        at com.google.android.gms.auth.zzd.getToken(Unknown Source)
        at com.google.android.gms.auth.zzd.getToken(Unknown Source)
        at com.google.android.gms.auth.GoogleAuthUtil.getToken(Unknown Source)
        at com.flowcrypt.email.api.email.protocol.OpenStoreHelper.openAndConnectToGimapsStore(OpenStoreHelper.java:77)
        at com.flowcrypt.email.api.email.protocol.OpenStoreHelper.openAndConnectToStore(OpenStoreHelper.java:172)
        at com.flowcrypt.email.api.email.sync.EmailSyncManager$BaseSyncRunnable.openConnectionToStore(EmailSyncManager.java:446)
        at com.flowcrypt.email.api.email.sync.EmailSyncManager$BaseSyncRunnable.runSyncTask(EmailSyncManager.java:475)
        at com.flowcrypt.email.api.email.sync.EmailSyncManager$ActiveSyncTaskRunnable.run(EmailSyncManager.java:574)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:428)
        at java.util.concurrent.FutureTask.run(FutureTask.java:237)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
        at java.lang.Thread.run(Thread.java:762)
```

**Comment by DenBond7** [comment]
ref https://mail.google.com/mail/u/0/#inbox/162d9032bce0676c

**Comment by DenBond7** [comment]
Fixed in #227
